test/WebSites/MvcTagHelpersWebSite/Views/MvcTagHelper_Home/Index.cshtml

```diff
@@ -45,9 +45,9 @@
         </a>
     </div>
     <div>
-        <a asp-route="areaRoute" asp-route-area="Customer"
-           asp-route-controller="Customer"
-           asp-route-action="Index"
+        <a asp-route="areaRoute" asp-route-area='"Customer"'
+           asp-route-controller='"Customer"'
+           asp-route-action='"Index"'
```

Comment:
Razor's handling of `string` values for `object` tag helper properties or dictionary entries is not great. Neither `"&quot;Customer&quot;"` nor `"\"Customer\""` work here; the first because this isn't an HTML context and the second because C# `string` parsing is, lets-call-it, minimal.  `"@("Customer")"` is the other way I could write this.